### PR TITLE
telebaton stun redux

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

PR #75 all over again

## Why It's Good For The Game

makes it consistent with other servers + way for synths and COs to defend themselves against angry marines

does NOT affect xeno combat

## Changelog

:cl:
add: the telebaton can now deal 50 stam damage when used on a human if not on harm intent
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
Good change +1

**Comment by JohnDoe50974** [comment]
Thanks :3

**Comment by stanalbatross** [comment]
> Make it deal stamina dmg, also, it can perma stun potentially a single target as it has no max energy...
> 
> Probably fine, but keep it in mind.

done

**Comment by morrowwolf** [comment]
So while two hits with a stunbaton will likely knock someone down given it is checking against energy armor (which most marines don't have) the telebaton is checking against melee defense which most marines do have a good bit of. It may be good to increase the stunforce higher to be more effective as it's three hits with just a regular USCM uniform to knock over and more with any armor. 125 stunforce will reliably knock over a marine in normal armor in two hits. 100 is three hits with standard gear/armor.

Having the extension and collapsing use danger and notice respectively rather than the same feels funny to me but designer choice and this PR doesn't explicitly touch that.

Otherwise, LGTM.

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 1]
```diff
@@ -75,14 +83,28 @@
...
 
 	if(blood_overlay && blood_color)
 		overlays.Cut()
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/carbon/human/target, mob/living/user)
+	if(target.check_shields(src, 0, "[user]'s [name]"))
+		return FALSE
+	user.visible_message(SPAN_DANGER("[user] knocks down [target] with \the [src]!"),\
```
Comment (JeserTheFox) [inline code/game/objects/items/weapons/swords_axes_etc.dm:97]: This always says "knocks down" even if you don't actually knock down target. Either need to be made more universal or do a check post-hit whether target is stunned/lying and show respective message.

**Review by morrowwolf — APPROVED** [review]
LGTM

**Pull request merged**